index.bs

```diff
@@ -798,8 +798,12 @@ To <dfn>queue a report for delivery</dfn> given an [=attribution report=] |repor
 
 To <dfn>serialize an [=attribution report=]</dfn> |report|, run the following steps:
 
+1. Let |destination| be |report|'s [=attribution report/attribution destination=].
+1. Assert: |destination| is not an [=opaque origin=].
 1. Let |data| be a [=map=] of the following key/value pairs:
 
+    : "`attribution_destination`"
+    :: The [=string/concatenate|concatenation=] of «|destination|'s scheme, "`://`", |destination|'s host».
```

Comment:
I think this should |destination|'s host, serialized to match the URL serialization algo: https://url.spec.whatwg.org/#url-serializing

However it seems like that definition is not exported, so it may be necessary to create an origin and serialize that directly.